test/functional/readconcern_tests.js

```diff
@@ -37,6 +37,10 @@ describe('ReadConcern', function() {
         var collection = db.collection('readConcernCollection');
         // Validate readConcern
         test.deepEqual({ level: 'local' }, collection.s.readConcern);
+        // create a collection using createCollection
```

Comment:
This test looks good! Let's keep the existing test as-is and move this new code into its own test.